src/rgw/rgw_common.cc

```diff
@@ -85,6 +85,7 @@ rgw_http_errors rgw_http_s3_errors({
     { EPERM, {403, "AccessDenied" }},
     { ERR_SIGNATURE_NO_MATCH, {403, "SignatureDoesNotMatch" }},
     { ERR_INVALID_ACCESS_KEY, {403, "InvalidAccessKeyId" }},
+    { ERR_INVALID_KMS_KEY, {403, "InvalidKmsKeyId" }},
```

Comment:
i don't see `InvalidKmsKeyId` listed in https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html, where does this come from?

i think we should just use `EINVAL` for these instead

Reply:
The "InvalidKmsKeyId" I added is not in https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html, in this case EINVAL is more reasonable, so I have Modified him as you suggested, thanks.